Software/facial_recognition/recognize.py

```diff
@@ -30,8 +31,27 @@ def recognize(classifier, trainer):
     # to its index in the array. Ex) Ozayr -> ID 0
     names = ["Ozayr"] 
 
+    # Check if the user has up to 3 cameras
+    def returnCameraIndexes():
+        index = 0
+        arr = []
+        while index < 3:
+            cap = cv2.VideoCapture(index)
+            if cap.isOpened() == True:
+                arr.append(index)
+            cap.release()
+            index += 1
+        cam_id = int(input(f"{len(arr)} cameras available. Enter a camera ID {arr}: "))
+        return cam_id
```

Comment:
im not to sure what the standard is for python but I think you should move the function definition outside of recognize. Will make the code cleaner

Reply:
Done